lighthouse-core/test/runner-test.js

```diff
@@ -276,47 +345,19 @@ describe('Runner', () => {
       .then(_ => {
         assert.ok(false);
       }, err => {
-        assert.ok(/The config must provide passes/.test(err.message));
+        assert.ok(/the config has defined no audits to evaluate/.test(err.message));
       });
   });
 
-  it('accepts existing auditResults', () => {
-    const url = 'https://example.com';
-    const config = new Config({
-      auditResults: [{
-        name: 'content-width',
-        rawValue: true,
-        score: true,
-        displayValue: '',
-      }],
-
-      categories: {
-        category: {
-          name: 'Category',
-          description: '',
-          audits: [
-            {id: 'content-width', weight: 1},
-          ],
-        },
-      },
-    });
-
-    return Runner.run(null, {url, config, driverMock}).then(results => {
-      // Mostly checking that this did not throw, but check representative values.
-      assert.equal(results.initialUrl, url);
-      assert.strictEqual(results.audits['content-width'].rawValue, true);
-    });
-  });
-
   it('returns reportCategories', () => {
     const url = 'https://example.com';
     const config = new Config({
-      auditResults: [{
-        name: 'content-width',
-        rawValue: true,
-        score: true,
-        displayValue: 'display',
+      passes: [{
+        gatherers: ['viewport-dimensions'],
       }],
+      audits: [
```

Comment:
how does this test audit output without running a gatherer or audit?

Reply:
it does run a gatherer and an audit. :o